source/manual/document-collection-emails-a-special-case.html.md

```diff
@@ -0,0 +1,72 @@
+---
+owner_slack: "#govuk-developers"
+title: Document Collection emails - a special case
+section: Emails
+type: learn
+layout: manual_layout
+parent: "/manual.html"
+---
+
+[Document Collections][document_collections_schema] are published by Whitehall, and rendered by [government-frontend][government-frontend]. The majority of document_collection pages allow users to sign up to email notifications via a [single page notification button][single_page_notification_button_component_guide]. Clicking this button subscribes users to email notifications when a major change is published for the page, or content reverse linked to the page.
+
+For a small number of document_collections published in late 2023, the email signup feature works differently. These differences were agreed as part of negotiations with a specific government department to unblock the retirement of specialist topics.
```

Comment:
What do you think about adding a short paragraph explaining the removal of specialist topics? People in the future might not know what specialist topics were. Something like (please, improve):

> In 2024 we removed specialist topics. It was the third tagging system for our documents, besides browse pages and taxonomy.  Besides other things, it involved moving the mailing lists registered for specialist topics to taxons and document collections (explained above).

Reply:
Good idea. I'm working on some documentation for that and so can add a link